templates/views/team.html

```diff
@@ -0,0 +1,162 @@
+                    </div>
+                    <div class="__shecodeteammember_text">
+                        <h5 class="name">Ada Nduka, OYOM</h5>
+                        <p class="role">Founder</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-md-4 col-sm-6 col-xs-12">
+                <div class="__shecodeteammember_box" data-toggle="modal" data-target="#team_modal">
+                    <div class="__shecodeteammember_img">
+                        <img src="/images/hair-2537564_1920.png" alt="" class="img-responsive">
+                    </div>
+                    <div class="__shecodeteammember_text">
+                        <h5 class="name">Ada Nduka, OYOM</h5>
+                        <p class="role">Founder</p>
+                    </div>
+                </div>
+            </div>
+
+
+        </div>
+    </div>
+
+    <!-- Modal -->
+    <div id="team_modal" class="modal fade team_modal" tabindex="-1" role="dialog" aria-labelledby="contactModalLabel" aria-hidden="true">
+        <div class="modal-dialog modal-dialog-centered">
+
+            <div class="modal-content ">
+
+
+                <div class="modal-body">
+                    <div class="row">
+                        <div class="col-sm-6">
+                            <div class="__shecodeteammember_box">
+                                <div class="__shecodeteammember_img">
+                                    <img src="/images/hair-2537564_1920.png" alt="" class="img-responsive">
+                                </div>
+                                <div class="__shecodeteammember_text">
+                                    <h5 class="name">Ada Nduka, OYOM</h5>
+                                    <p class="role">Founder</p>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-sm-6">
+                            <div class="__shecodeteammember_details">
+                                <span class="close" data-dismiss="modal" aria-hidden="true" title="close">x</span>
+                                <br>
+                                <p>With over 5 years of hands-on experience, Akolade has designed several digital products spanning the oil & gas, energy, education, healthcare, and entertainment industries. Akolade holds a bachelor’s degree in mechanical
+                                    engineering from Coventry University with particular distinctions in product innovation and design principles.
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <!-- End of Modal body -->
+            </div>
+
+            <!-- End of Modal content -->
+        </div>
+        <!-- End of Modal dialog -->
+    </div>
+    <!-- End of Modal -->
+
+</section>
+{% endblock %}
```

Comment:
Missing new line at the end of file

Reply:
Firstly, the !important is used on the different screens styling for responsiveness. 
"Missing new line at the end of file" I don't understand this.